SH4RE/SH4RE/Account/ProfileView.swift

```diff
@@ -15,6 +15,7 @@ struct ProfileView: View {
     @State private var name:String = "" // if not name
     @State private var numberOfStars:Float = 4.0
     @StateObject private var listingsView = ListingViewModel()
+    @State var allReviews = [Review]()
```

Comment:
i still do not really understand this method of initializing variables, does it not place something in there when you do this? I am so confusion

Reply:
It just specifies what that list holds. In this case the list holds Reviews

Reply:
@bbryanb333 I find the syntax to be a little confusing to read because it is not type-declared like all the rest of our variables are. In the future and to align with the way we initialize other variables, would you be able to do this instead?
```@State private var allReviews: [Review] = []```